pkg/ipam/poolallocator/allocator.go

```diff
@@ -384,3 +391,21 @@ func (a *IPPoolAllocator) HasContainer(containerID string) (bool, error) {
 	}
 	return false, nil
 }
+
+func (a IPPoolAllocator) Used() int {
+	_, allocators, _ := a.readPoolAndInitIPAllocators()
+	used := 0
+	for _, allocator := range allocators {
+		used += allocator.Used()
+	}
+	return used
+}
+
+func (a IPPoolAllocator) Total() int {
+	_, allocators, _ := a.readPoolAndInitIPAllocators()
+	total := 0
+	for _, allocator := range allocators {
+		total += allocator.Total()
```

Comment:
Instead of reading the pool and init the allocators another time, could we use the existing `allocators` to get `Total` directly, maybe pass it as an argument of `appendPoolUsage` and `removePoolUsage`? And I think you have added `ToTal` method to `MultiIPAllocator`, we could call it directly?